build/teamcity-check.sh

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env bash
 set -exuo pipefail
+
```

Comment:
why the double newlines here and in acceptance? everywhere else only one is used.